styles/styles.css

```diff
@@ -224,6 +224,24 @@ html {
   align-items: center;
 }
 
+/* Modifiers */
+
+.nc--light-on-dark,
```

Comment:
This should be `.nc-light-on-dark` since there's no `.nc` block to modify here, so it's considered a block in its own right 👍